ajax/libs/multi.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "multi.js",
+  "filename": "multi.min.js",
+  "version": "0.2.3",
+  "description": "A user-friendly replacement for select boxes with multiple attribute enabled",
+  "keywords": [
+    "user-friendly",
+    "select",
+    "multiple",
+    "simple"
```

Comment:
Let's also add `select boxes` & `search`.
Because it's a select boxes, also it provides search functionality.
Thanks.